ansible/provision-pool-play/examples/roks-pool-vars.yml

```diff
@@ -0,0 +1,4 @@
+---
+# PLACE HOLDER ONLY
+"roks_api_token": "3-rKI5k7QG10Dq0ycSGiPHxb6hqfXmbeybahqshO-fI"
```

Comment:
did you mean to include the token?

Reply:
Came from copy of provision-ocp-cluster play, I did nathing to add this!